devicetypes/Orvibo/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy

```diff
@@ -0,0 +1,60 @@
+/**
+ *  Copyright 2015 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Orvibo Contact Sensor test", namespace: "smartthings", author: "SmartThings",  vid:"generic-contact-3", ocfDeviceType: "x.com.st.d.sensor.contact") {
+        capability "Contact Sensor"
+        capability "Battery"
```

Comment:
Consider adding capability "Refresh" to sync device state in case of missing events etc